code-security/admin_guide/scan-monitor/secrets-scanning.adoc

```diff
@@ -0,0 +1,52 @@
+== Secrets Scanning
+
+You can use Code Security to detect and remove secrets in IaC files stored in your IDEs, Git-based VCS, and CI/CD pipelines.
+
+A secret is a programmatic access key that provides systems with access to information, services or assets. Developers use secrets such as API keys, encryption keys, OAuth tokens, certificates, PEM files, passwords, and passphrases to enable their application to securely communicate with other cloud services. 
+
+For identifying secrets, Prisma Cloud provides default policies that use domain-specific and generic syntax to match on specific signatures and patterns to validate the likelihood or entropy of a string being a secret. You can view the scan aresults directly on *Code Security > Projects* or on the CLI if using Checkov.
+
+The following file types or extensions are scanned for secrets:
```

Comment:
I thought we weren't going to include this because it will change in a few weeks.

Reply:
we were not listing the policies.

Reply:
Ah ok. This works.